This is a wdspec test.
[FAIL] test_params_request_invalid_phase[beforeRequestSent]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_request_invalid_phase[responseStarted]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_request_no_such_request_after_cancel
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_request_no_such_request_after_provideCredentials
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_invalid_type[None]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_invalid_type[False]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_invalid_type[42]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_invalid_type[value3]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_invalid_type[value4]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_invalid_value[]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_invalid_value[foo]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_invalid_credentials[missing username]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_invalid_credentials[missing password]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_invalid_credentials[missing username and password]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_invalid_credentials[missing type]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_invalid_credentials[missing credentials]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_type_invalid_type[None]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_type_invalid_type[False]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_type_invalid_type[42]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_type_invalid_type[value3]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_type_invalid_type[value4]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_type_invalid_value[]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_type_invalid_value[foo]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_username_invalid_type[None]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_username_invalid_type[False]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_username_invalid_type[42]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_username_invalid_type[value3]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_username_invalid_type[value4]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_password_invalid_type[None]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_password_invalid_type[False]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_password_invalid_type[42]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_password_invalid_type[value3]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
[FAIL] test_params_action_provideCredentials_credentials_password_invalid_type[value4]
  TypeError: add_intercept() got an unexpected keyword argument 'contexts'
Harness: the test ran to completion.
